.prettierignore

```diff
@@ -0,0 +1,2 @@
+.prettierignore
```

Comment:
You'll probably want to ignore generated JS files as well, see the list in the sdk-node repo: https://github.com/temporalio/sdk-node/blob/main/.prettierignore